### PR TITLE
adding a few more custom css options too the documentation 

### PR DESCRIPTION
**Pull request by blassley**
*Thank you for contributing to Homarr! So that your Pull Request can be handled effectively, please populate the following fields (delete sections that are not applicable)*

### Category
documentation

### Overview
adding blur and border custom css

### CODE REVIEW

**Comment by manuel-rw** [comment]
Thank you for your contribution, but can you please stop opening new pull requests the entire time?
You opened already 4 with the same content. Please do not open more!
Please mention me when you're done and this can be reviewed.

**Comment by blassley** [comment]
> Thank you for your contribution, but can you please stop opening new pull requests the entire time?
> 
> You opened already 4 with the same content. Please do not open more!
> 
> Please mention me when you're done and this can be reviewed.

Yes of course I'm not sure if this was due to connection issue earlier or what but visual code kept failing then I looked on GitHub and they were all here lol take a look at this when you have time

File: docs/customizations/custom-css.md [hunk 1]
```diff
@@ -22,6 +22,43 @@ After entering your rules, click on the "Submit" button to save your changes. Yo
...
+}
+```
+Adding the above to your custom CSS will give you blur support for both desktop and mobile. You can change how much blur is wanted by changing the (60px) to a choice that better suits your needs. 
+
+![homarr custom css results](./img/customizations-custom-css-blur.png)
+
+We can also add blur support for the settings modal. Here is an example of how we achieve this.
+
+```
+.mantine-w4dwz9 {
```
Comment (manuel-rw) [inline docs/customizations/custom-css.md:40]: Can you mention, that these class names will change each time Homarr is updated?
Also, you could create a ticket for this, so we crate unused static class names, so they don't change anymore.

**Review by manuel-rw — CHANGES_REQUESTED** [review]
Almost good to go. Thank you for your work!

**Comment by manuel-rw** [comment]
Thanks for the contribution!

**Pull request merged**